sdk/nodejs/tests/sxs_ts_latest/package.json

```diff
@@ -3,7 +3,7 @@
     "version": "${VERSION}",
     "license": "Apache-2.0",
     "dependencies": {
-        "@pulumi/pulumi": "latest"
+        "@pulumi/pulumi": "=1.6.0"
```

Comment:
note: i need to lock against pulumi `1.6` because `1.7.1` is *not* compatible with `1.7.0` (because of this very issue we are fixing).  Effectively 1.7.0 is hte screwup one that we need to overlook.  Once 1.7.1 is in, we can unlock this and validate that future versions are compatible with 1.7.1